pkg/agent/cniserver/server_test.go

```diff
@@ -12,10 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build linux
-// +build linux
-
-// TODO: fix the CNI ADD test for Windows.
+//go:build linux || windows
+// +build linux windows
```

Comment:
why don't just remove them?

Reply:
It can not run on darwin or other platform except for linux and windows, this is because some dependent logic functions in this path have implementations only on Linux and Windows.

Reply:
Isn't it the same for server.go? I assume this package cannot be compiled and tested on darwin anyway with or without the build tag.

Reply:
I used to have a try to remove the tags, and saw that this test file could be run on darwin with errors ( although I triggered it actively ). 

Before my patch, the previous version of `server_test.go` also has explicit tag to require it to run on Linux only. In my patch, I just want to ensure some general cases could be run on both Linux and Windows, so I modify it.

Reply:
Removed the marks in the latest update. There is no unit-test run darwin in the github actions, cases are passed.